src/main/java/worldofzuul/DomainLayer/Room.java

```diff
@@ -45,15 +45,14 @@ public void setExit(String direction, Room neighbor) {
         exits.put(direction, neighbor);
     }
 
-    public void setCanCheckout(boolean checkout){
-        this.canCheckout = checkout;
+    public void addCashier(int xPosition, int yPosition){
+        cashiers.add(new Cashier(xPosition, yPosition));
     }
 
-    public boolean canCheckout(){ //boolean to verify if it's possible to checkout in the current room.
-        return canCheckout;
+    public boolean canCheckout(){
+        return false;
     }
```

Comment:
Det går ikke man bare returnerer `false`, for så virker det ikke længere for CLI'en.
```suggestion
    public boolean canCheckout(){
        return this.cashiers.size() > 0;
    }
```

Reply:
havde ændret det, havde bare glemt at push igen